### ISSUE TITLE
NoClassDefFoundError: org/gradle/logging/ConsoleRenderer when trying to use the plugin

### ISSUE THREAD

**Issue by centic9** [konrad-garus/classycle-gradle-plugin#2]
I tried to use the plugin in Gradle 2.14.1, but it fails 

In the build.gradle I have:
```
buildscript {
	repositories {
		repositories { maven { url "https://plugins.gradle.org/m2/" } }
	}

	dependencies {
		classpath "gradle.plugin.pl.squirrel:classycle-gradle-plugin:1.1"
	}
}
apply plugin: 'pl.squirrel.classycle'
```

I get:
```
* What went wrong:
Execution failed for task ':shared:elasticsearch:classycleTest'.
> org/gradle/logging/ConsoleRenderer

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
...
Caused by: java.lang.NoClassDefFoundError: org/gradle/logging/ConsoleRenderer
        at pl.squirrel.classycle.ClassyclePlugin.lambda$null$0(ClassyclePlugin.java:62)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:621)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:604)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
        ... 60 more
Caused by: java.lang.ClassNotFoundException: org.gradle.logging.ConsoleRenderer
        ... 65 more
```

Not sure how a base class of Gradle itself can be missing, is the plugin requiring some specific version of Gradle?

**Comment by konrad-garus** [comment]
It's been a while (and a few Gradle versions) since I've updated it. Looks like it is time to have a look!

**Comment by oliver-hofmann** [comment]
This looks like the problem from [this Gradle discussion](https://discuss.gradle.org/t/gradle-2-14-breaks-plugins-using-consolerenderer/18045)

Interestingly I had no problem running this plugin with Gradle 3.1 and 3.2 as long as classycle can do its work (Groovy does not seem to resolve the logging class as long as there's nothing to log).

However, I run into the very same problem when including spring-boot-configuration-processor into my spring boot project. Backporting it to Gradle 2.13 showed the real problem in my case (might be an issue on its own): 
`Execution failed for task ':classycleMain'.
Classycle check failed: java.lang.RuntimeException: java.io.IOException: ...\build\classes\main\META-INF\spring-configuration-metadata.json is an invalid file.. See report at ...`

So the NoClassDefFoundError may hide the real problem which cannot be logged using newer gradle versions.

**Comment by oliver-hofmann** [comment]
This Patch may solve the Gradle compatibility issue:
[remove-ConsoleRenderer-usage.zip](https://github.com/konrad-garus/classycle-gradle-plugin/files/622343/remove-ConsoleRenderer-usage.zip)

**Comment by konrad-garus** [comment]
Fixed in 6431c35cd84ac2d394996d29ad6b642a0cd15631, deployed plugin version 1.2. Thanks guys, you're awesome!
